index.js

```diff
@@ -0,0 +1,58 @@
+// QUESTION: 1
+
+const sumOf = (num) => {
+
+  if(num === 0 )return 0
+
+  return num + sumOf(num -1)
+
+}
+console.log(sumOf(6));
+
+// QUESTION: 2
+
+const multArr = (arr)  => {
+  if(arr.length === 0) return 1
+  let el = arr.pop()
+  return multArr(arr) * el
+}
+
+console.log(multArr([5,5,1,2]));
+
+// QUESTION: 3
+
+const concatArr = (arr) => {
+  if(arr.length === 0) return ""
+    let str = arr.pop()
+    return concatArr(arr) + "" + str + " "
+}
+
+console.log(concatArr(['or', 'just', 'the', 'end', 'of', 'time']))
+
+// QUESTION: 4
+const sumEvens = (arr) => {
+  if(arr.length === 0 )return 0
+  let el = arr.pop()
+  if(el % 2 === 0){
+    return sumEvens(arr) + el
+}else {
+  return sumEvens(arr) + 0
```

Comment:
you can just return `sumEvens(arr)` on line 39 since the 0 doesn't add anything anyway